Files/Interacts/Interaction.cs

```diff
@@ -1182,5 +1187,19 @@ public void PushJumpChar(char letter)
         {
             App.CurrentInstance.ViewModel.JumpString += letter.ToString().ToLower();
         }
+
+        public async Task<string> GetHashForFile(ListedItem fileItem, string nameOfAlg)
```

Comment:
Can you rename this to `GetMD5HashFromFile`?

Reply:
Actually, if we pass in an alternative second parameter, we can get different types of hashes.

Reply:
That's why I left it as a generic name just in case.

Reply:
OK, that is very good.